src/schema/objects/metadata.yaml

```diff
@@ -561,6 +561,53 @@ DecayCorrectionFactor:
   type: array
   items:
     type: number
+DeidentificationMethod:
+  name: DeidentificationMethod
+  display_name: Deidentification Method
+  description: |
+    A description of the mechanism or method used to remove the Patient's identity.
+    Corresponds to DICOM Tag 0012, 0063 `De-identification Method`.
+  type: string
+DeidentificationMethodCodeSequence:
+  name: DeidentificationMethodCodeSequence
+  display_name: Deidentification Method Code Sequence
+  description: |
+    A sequence of code objects describing the mechanism or method use to remove the Patient's identity.
+    Corresponds to DICOM Tag 0012, 0064 `De-identification Method Code Sequence`.
+  type: array
+  items:
+    type: object
+    recommended_fields:
+      - CodeValue
+      - CodeMeaning
+      - CodingSchemeDesignator
+      - CodingSchemeVersion
+    properties:
+      CodeValue:
+        name: CodeValue
+        type: string
+        description: |
+          An identifier that is unambiguous within the Coding Scheme
+          denoted by Coding Scheme Designator and Coding Scheme Version.
+          Corresponds to DICOM Tag 0008, 0100 `Code Value`.
+      CodeMeaning:
+        name: CodeMeaning
+        type: string
+        description: |
+          Text that has meaning to a human and conveys the meaning of the term
+          Corresponds to DICOM Tag 0008, 0104 `Code Meaning`.
+      CodingSchemeDesignator:
+        name: CodingSchemeDesignator
+        type: string
+        description: |
+          The identifier of the coding scheme in which the coded entry is defined.
+          Corresponds to DICOM Tag 0008, 0102 `Coding Scheme Designator`.
+      CodingSchemeVersion:
+        name: CodingSchemeVersion
+        type: string
+        description: |
+          An identifier of the version of the coding scheme if necessary to resolve ambiguity.
+          Corresponds to DICOM Tag 0008, 0103 `Coding Scheme Version`.
```

Comment:
I think this is complicated enough warranting addition to https://github.com/bids-standard/bids-examples if not as an example here in the BIDS spec text.  Do we have some nice example DICOMs to check those fields out?

Reply:
I think if someone were creating these tags from scratch at the BIDS level, they would use DeidentificationMethod rather than DeidentificationMethodCodeSequence. The latter is more provided to allow direct translation by dcm->nii and dcm->bids software. 

I'm attaching some nice example DICOM, as requested. This is MRI of a spaghetti squash (thanks @captainnova), and I've further de-identified it, so there's no PHI. I included the tags as if I had de-faced it, but it is a squash, so there was no face to de-face. 

[dcm_fruit_deidentified.zip](https://github.com/bids-standard/bids-specification/files/15122386/dcm_fruit_deidentified.zip)


The relevant dcm tags look like this:
```
(0012, 0063) De-identification Method            LO: ['mri_reface 0.3.4', 'Per DICOM PS 3.15 AnnexE. Details in 0012,0064']
(0012, 0064)  De-identification Method Code Sequence   5 item(s) ---- 
   (0008, 0100) Code Value                          SH: '113102'
   (0008, 0102) Coding Scheme Designator            SH: 'DCM'
   (0008, 0103) Coding Scheme Version               SH: '01'
   (0008, 0104) Code Meaning                        LO: 'Clean Recognizable Visual Features Option'
   ---------
   (0008, 0100) Code Value                          SH: 'replace_recognizable'
   (0008, 0102) Coding Scheme Designator            SH: 'mri_reface'
   (0008, 0103) Coding Scheme Version               SH: '0.3.4'
   (0008, 0104) Code Meaning                        LO: 'Replace face, ears, and artifacts in air'
   ---------
   (0008, 0100) Code Value                          SH: '113100'
   (0008, 0102) Coding Scheme Designator            SH: 'DCM'
   (0008, 0104) Code Meaning                        LO: 'Basic Application Confidentiality Profile'
   ---------
   (0008, 0100) Code Value                          SH: '113107'
   (0008, 0102) Coding Scheme Designator            SH: 'DCM'
   (0008, 0104) Code Meaning                        LO: 'Retain Longitudinal Temporal Information Modified Dates Option'
   ---------
   (0008, 0100) Code Value                          SH: '113111'
   (0008, 0102) Coding Scheme Designator            SH: 'DCM'
   (0008, 0104) Code Meaning                        LO: 'Retain Safe Private Option'
   ---------
```


With https://github.com/rordenlab/dcm2niix/pull/813, the relevant portion of json output is:
```
        "DeidentificationMethod": "mri_reface 0.3.4\\Per DICOM PS 3.15 AnnexE. Details in 0012,0064",
        "DeidentificationMethodCodeSequence": [ 
          { 
                "CodeValue": "113102",
                "CodingSchemeDesignator": "DCM",
                "CodingSchemeVersion": "01",
                "CodeMeaning": "Clean Recognizable Visual Features Option"
          },
          { 
                "CodeValue": "replace_recognizable",
                "CodingSchemeDesignator": "mri_reface",
                "CodingSchemeVersion": "0.3.4",
                "CodeMeaning": "Replace face, ears, and artifacts in air"
          },
          { 
                "CodeValue": "113100",
                "CodingSchemeDesignator": "DCM",
                "CodeMeaning": "Basic Application Confidentiality Profile"
          },
          { 
                "CodeValue": "113107",
                "CodingSchemeDesignator": "DCM",
                "CodeMeaning": "Retain Longitudinal Temporal Information Modified Dates Option"
          },
          { 
                "CodeValue": "113111",
                "CodingSchemeDesignator": "DCM",
                "CodeMeaning": "Retain Safe Private Option"
          }
        ],
```